share/spice/coderwall/coderwall.js

```diff
@@ -58,8 +52,32 @@
             },
 
             templates: {
-                group: 'icon'
+                group: 'icon',
+                variants: {
+                    iconImage: 'large'
+                },
+                options: {
+                    content: DDH.coderwall.content
+                }
             }
         });
     };
+
+    Handlebars.registerHelper("parseMD", function(content){
+        var contentArr = [];
+        if (!content) return;
+        else content.split(/\n+/).forEach(function(line) {
+            // exclude lines that are only whitespace characters
+            if (/[a-z0-9]/.test(line)) {
+                line = line.replace(/\*\*(.+?)\*\*/g, '<strong>$1</strong>') // bold
+                  .replace(/(?:_(.+?)_|\*(.+?)\*)/g, '<em>$1$2</em>') // italicize
+                  .replace(/`(.+?)`/g, '<code>$1</code>') // monospace
+                  .replace(/~~(.+?)~~/g, '<del>$1</del>') // strikethrough
```

Comment:
If you need any help with HTML escaping check out [MarkdownReference](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/markdown_reference/markdown_reference.js) - I found a pretty good helper for it a while back.